connectors/src/resources/zendesk_resources.ts

```diff
@@ -235,6 +256,21 @@ export class ZendeskBrandResource extends BaseResource<ZendeskBrand> {
     );
   }
 
+  static async fetchAllTickets({
```

Comment:
Shouldn't we have instead a `fetchByBrandId` in `ZendeskTicketResource`?

Reply:
Feels weird here to call directly the model ZendeskTicket and to create instances of ZendeskTicketResource, but  I might be wrong?

Reply:
Done, I moved around the methods to have each Resource take care of the underlying resource instead of having the BrandResource deal with tickets, articles and categories